src/WebJobs.Script.WebHost/WebHostServiceCollectionExtensions.cs

```diff
@@ -188,6 +188,7 @@ public static void AddWebJobsScriptHost(this IServiceCollection services, IConfi
             services.ConfigureOptions<LanguageWorkerOptionsSetup>();
             services.ConfigureOptionsWithChangeTokenSource<AppServiceOptions, AppServiceOptionsSetup, SpecializationChangeTokenSource<AppServiceOptions>>();
             services.ConfigureOptionsWithChangeTokenSource<HttpBodyControlOptions, HttpBodyControlOptionsSetup, SpecializationChangeTokenSource<HttpBodyControlOptions>>();
+            services.ConfigureOptionsWithChangeTokenSource<FunctionsHostingConfigOptions, FunctionsHostingConfigOptionsSetup, ConfigurationChangeTokenSource<FunctionsHostingConfigOptions>>();
```

Comment:
Is the `ConfigurationChangeTokenSource` necessary here? I thought that config and options reload will all be built into the file watcher from `FunctionsHostingConfigSource`.

Reply:
This was necessary to fire `OnChange `for `IOptionsMonitor `and after introducing the section it fires without the line - removed.